executor/write.go

```diff
@@ -886,15 +886,58 @@ func (e *InsertValues) getRows(cols []*table.Column, ignoreErr bool) (rows [][]t
 }
 
 func (e *InsertValues) getRow(cols []*table.Column, list []expression.Expression, ignoreErr bool) ([]types.Datum, error) {
-	vals := make([]types.Datum, len(list))
+	// row is a eval context, so `insert t set b = 1, a = b + 1` can eval correctly.
```

Comment:
I think this comment is not that explicit.
Can this comment and comments int lone 893 and 894 be removed,
and add a comment for this function.